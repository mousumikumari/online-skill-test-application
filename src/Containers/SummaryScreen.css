.container{
    background-color: antiquewhite;
    border-radius: 15px;
    padding:18px;
    text-align: center;
    box-shadow: 0 8px 6px -6px black;
      position: relative;
      max-width: 918px;
      margin: 0 auto;
      margin-top: 100px;
      border-style: groove;
      height: 500px;
  }