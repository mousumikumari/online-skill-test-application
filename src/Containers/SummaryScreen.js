import React from "react";

const summaryScreen = ({ location }) => {
  const { state } = location;
  return (
    <div className="container">
      <h4>Thank You! </h4>
      <p>
        <b>Name :</b> {state.firstName} {state.lastName}
        <br />
        <b>Email :</b> {state.email}
        <br />
        <b>Questions Attempted : </b>
        <span className="number-score">
          {state.questionsAttempted} / {state.questionCount}
        </span>
        <br />
        <b>Score :</b>{" "}
        <span className="number-score">
          {state.candidateScore} / {state.questionCount}
        </span>
      </p>
    </div>
  );
};

summaryScreen.propTypes = {
  location: Object.isRequired,
};

export default summaryScreen;
