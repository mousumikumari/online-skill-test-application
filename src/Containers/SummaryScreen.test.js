/* eslint-disable */
import React from "react";
import renderer from "react-test-renderer";
import SummaryScreen from "./SummaryScreen";

const location = {
  hash: "",
    key: "yuuhiji",
  pathname: "/SummaryScreen",
  search: "",
  state : {
    questionIndex: 0,
    isFalseLogin: false,
    quizTime: 600,
    questionCount: 0,
    IsQuizSubmitted: false,
    skillLevel: "",
    questionData: "",
    candidateDetails: [],
    candidateResponses: [],
    currentResponse: [],
    questionsAttempted: 0,
    candidateScore: 0,
    firstName: "",
    lastName: "",
    email: "",
    gender: "",
    password: "",
  },
}
test("First Submit Test Snapshot test", () => {
  const component = renderer.create(
    <SummaryScreen location= { location }/>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
