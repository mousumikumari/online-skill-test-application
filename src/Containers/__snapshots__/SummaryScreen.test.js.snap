// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First Submit Test Snapshot test 1`] = `
<div
  className="container"
>
  <h4>
    Thank You! 
  </h4>
  <p>
    <b>
      Name :
    </b>
     
    
     
    
    <br />
    <b>
      Email :
    </b>
     
    
    <br />
    <b>
      Questions Attempted : 
    </b>
    <span
      className="number-score"
    >
      0
       / 
      0
    </span>
    <br />
    <b>
      Score :
    </b>
     
    <span
      className="number-score"
    >
      0
       / 
      0
    </span>
  </p>
</div>
`;
